client/components/Dashboard/DashboardDumb.js

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { Container, Header, Grid, Button, Divider } from 'semantic-ui-react';
+import CardGroup from '../CardGroup/CardGroupDumb';
+import { firstSentenceOnly } from '../../../utils/index';
+
+const groups = [
+  {
+    id: 1,
+    name: 'Linear Algebra',
+    description: 'All things Linear Algebra! I love la.',
+  },
+  {
+    id: 2,
+    name: 'Calculus',
+    description: 'All things Calculus! I love calc.',
+  },
+  {
+    id: 3,
+    name: 'Discrete Math',
+    description: 'All things Discrete Math!',
+  },
+  {
+    id: 4,
+    name: 'Geometry',
+    description: 'All things Geometry!',
+  },
+  {
+    id: 5,
+    name: 'Algorithms',
+    description: 'All things Algorithms!',
+  },
+  {
+    id: 6,
+    name: 'Data Structures',
+    description: 'All things Data Structures!',
+  },
+];
+
```

Comment:
Now I know where the things came from in the snapshot you showed us in our slack channel.

Reply:
Yep! Just made some dummy data.